packages/react-core/src/demos/examples/TextInputGroup/AttributeValueFiltering.tsx

```diff
@@ -191,11 +195,11 @@ export const AttributeValueFiltering = () => {
   };
 
   /** close the menu when a click occurs outside of the menu or text input group */
-  const handleClick = event => {
+  const handleClick = (event: MouseEvent) => {
     if (
       menuRef.current &&
-      !menuRef.current.contains(event.target) &&
-      !textInputGroupRef.current.contains(event.target)
+      !menuRef.current.contains(event.target as Node) &&
```

Comment:
I'm not used to seeing this type. Does it work if it's `React.ReactNode`? Maybe that's what I'm more used to seeing?

Reply:
It may also be `HTMLElement`? It may not be wrong either - I think we just tend to be a little more specific...

Reply:
TypeScript complains about `React.ReactNode`, but `HTMLElement` does work since it's an extension of an extension of `Node`. Pushing this change shortly.